java/release_notes.txt

```diff
@@ -1,3 +1,12 @@
+Jul 14, 2016: libphonenumber-7.4.5
+Metadata changes:
+ - Updated phone metadata for region code(s): CN, HN, IN, KW, LV, TC
+ - Updated short number metadata for region code(s): AE, BR, GB, ZA
+ - New geocoding data for country calling code(s): 93 (fa), 992 (en), 1747 (en)
```

Comment:
Thanks for pointing this out.
Done in new commit.
https://github.com/googlei18n/libphonenumber/pull/1204/commits/e5920a415c9dbdb1ab42c1ef9a6ae2f5583302da